clients/src/main/java/org/apache/kafka/common/TopicPartitionInfo.java

```diff
@@ -79,9 +95,24 @@ public List<Node> isr() {
         return isr;
     }
 
+    /**
+     * Return the eligible leader replicas of the partition. Note that the ordering of the result is unspecified.
+     */
+    public List<Node> elr() {
```

Comment:
Would you consider using the full name for the uninitiated?

```suggestion
    public List<Node> eligibleLeaderReplicas() {
```